public/resources/core/js/services/product-offering.service.js

```diff
@@ -31,7 +31,7 @@
         .module('app')
         .factory('Offering', ['$q', '$resource', 'URLS', 'LIFECYCLE_STATUS', 'User', 'ProductSpec', 'Category', ProductOfferingService]);
 
-    function ProductOfferingService($q, $resource, URLS, LIFECYCLE_STATUS, User, ProductSpec, Category) {
+    function ProductOfferingService($q, $resource, URLS, CHARGE_PERIODS, CURRENCY_CODES, TAX_RATE, LIFECYCLE_STATUS, User, ProductSpec, Category) {
```

Comment:
This is not longer going to work, you can see that the injection model has changed to the array one, this has been done to enable js minimization for production